pyproject.toml

```diff
@@ -48,6 +48,7 @@ entry-points."compliance_checker.suites"."acdd-1.3" = "compliance_checker.acdd:A
 entry-points."compliance_checker.suites"."cf-1.6" = "compliance_checker.cf.cf:CF1_6Check"
 entry-points."compliance_checker.suites"."cf-1.7" = "compliance_checker.cf.cf:CF1_7Check"
 entry-points."compliance_checker.suites"."cf-1.8" = "compliance_checker.cf.cf:CF1_8Check"
+entry-points."compliance_checker.suites"."cf-1.9" = "compliance_checker.cf.cf:CF1_9Check"
```

Comment:
👍🏽 

We should also update the README to reflect that.

Reply:
Where in README do you think this should go?  I think in general there should be more documentation for developing plugins, but current README doesn't really cover development workflow.

Reply:
The main table, that describes the checks, is stuck in CF 1.8. That README file is also used to build the docs. I can send a PR for it later today.